pkgs/development/python-modules/pyside/tools.nix

```
@@ -13,7 +13,8 @@ buildPythonPackage rec {
 
   enableParallelBuilding = true;
 
-  buildInputs = [ cmake pyside qt4 pysideShiboken ];
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ pyside qt4 pysideShiboken ];
 
   meta = {
```

```suggestion
  meta = with lib; {
```

Now `lib` will be in scope so you won't need `lib.*` below.